src/Grid.hpp

```diff
@@ -0,0 +1,90 @@
+//
+//  Grid.hpp
+//  Game of Life
+//
+//  Created by Alex Frasca on 01/05/2023.
+//
+#pragma once
+#ifndef Grid_hpp
+#define Grid_hpp
+
+#include <SFML/Graphics.hpp>
+#include <vector>
+#include <cmath>
+#include <functional>
+
+namespace conway {
+
+//several values so we can draw nice colors as the cells die instead of just on/off
+enum CELL_STATE {
+    CELL_OFF = 0,
+    CELL_LOW = 1,
+    CELL_MEDIUM = 2,
+    CELL_HIGH = 3,
+    CELL_ON = 4
+};
+const float DEFAULT_SPACING = 16.0;
+
+struct Cell {
+    
+    Cell() {};
```

Comment:
This struct should follow the Rule of Zero, it doesn't need custom constructors

Reply:
I just removed the `<Cell() {}>` here. It can still have a constructor with parameters, right?

Reply:
It doesn't need any, users can use braced or designated init to initialize specific members.